test/inductor/test_torchinductor.py

```diff
@@ -6471,10 +6471,18 @@ def test_log_fp64(self):
         def fn(x):
             return torch.log(x), torch.log2(x)
 
-        self.common(
-            fn,
-            (torch.randn([1024], dtype=torch.float64) + 10,),
+        device_interface = get_interface_for_device(self.device)
+        _dtype = torch.float64
+        ctx = (
+            contextlib.nullcontext()
+            if device_interface.is_dtype_supported(_dtype)
```

Comment:
I though I've added a convenience method to take care of that
```suggestion
            if self.is_dtype_supported(_dtype)
```